### DISCUSSION TITLE
内容过滤误报

### DISCUSSION THREAD

**Discussion by sparkssssssss** [BurnHair/docs#209] [技术支持]
![image](https://github.com/user-attachments/assets/71796da6-dbcc-42ff-800e-24bd4a2fe6d3)
没联想到哪里nfsw了,感觉是误判了

**Comment by BennyThink** [comment]
你试试开一个新的会话看看，我等会也改改阈值吧

**Comment by sparkssssssss** [comment]
新开可以的
历史会话里面也都是之前api给的,难道有些字符经过排列组合,有了奇怪的意思.
带佬看看优化下吧,

**Reply by BennyThink** [reply]
嗯 可能是之前有命中的，我已经调整了阈值
